examples/cisco/demo_interface.pp

```diff
@@ -108,24 +108,39 @@
       warning('This platform does not support cisco_bridge_domain')
     }
 
-    # For private vlan
+    # Private-vlan
     if platform_get() =~ /n(3|5|6|7|9)k/ {
       cisco_vlan { '445':
-        ensure     => present,
-        private_vlan_type => 'isolated',
+        ensure                              => present,
+        #pvlan_type                          => 'isolated',
+        private_vlan_type => 'isolated',     # DEPRECATED
```

Comment:
I was waiting for Sai's commit #325. I just merged that so I'll update this now.